doc/admin-guide/files/ssl_server_name.config.en.rst

```diff
@@ -0,0 +1,124 @@
+:ts:cv:`proxy.config.ssl.servername.filename`.
+
+The configuration file is Lua based. After parsing and executing the Lua code, the global Lua variable
+``server_config`` is examined. Its value should be a list of tables. Each table is set of key / value
+pairs that create a configuration item.
+
+======================= ==============================================================================
+Key                     Meaning
+======================= ==============================================================================
+fqdn                    Fully Qualified Domain Name. This item is used if the SNI value matches this.
+
+verify_origin_server    One of the values :code:`NONE`, :code:`MODERATE`, or :code:`STRICT`.
+                        By default this is ::ts:cv:`proxy.config.ssl.client.verify.server`.
+
+verify_client           One of the values :code:`NONE`, :code:`MODERATE`, or :code:`STRICT`.
+                        By default this is :ts:cv:`proxy.config.ssl.client.certification_level`.
+
+client_cert             The client certificate to use for the outbound connection. If this is relative
+                        it is relative to the path in :ts:cv:`proxy.config.ssl.server.cert.path`.
+                        If not set :ts:cv:`proxy.config.ssl.client.cert.filename` is used.
+
+disable_h2              :code:`true` or :code:`false`. If :code:`false` then HTTP/2 is removed from
+                        the valid next protocol list. It is not an error to set this to :code:`false`
+                        for proxy ports on which HTTP/2 is not enabled.
+
+tunnel_route            Destination as an FQDN and port, separated by a colon '``:``'.
+======================= ==============================================================================
+
+Client verification, via ``verify_client``, correponds to setting
+:ts:cv:`proxy.config.ssl.client.certification_level` for this connection as noted below.
+
+:code:`NONE` -- ``0``
+   Do not request a client certificate.
+
+:code:`MODERATE` - ``1``
+   Request a client certificate and do verification if one is provide, but
+   do not enforce verification. A failure is logged but the connection is
+   allowed.
+
+:code:`STRICT` - ``2``
+   Request a client certificate and require one to be provided and verified.
+   A verification failure will cause the connection to fail.
+
+Upstream (server) verification is similar to client verification except there is always an upstream certificate.
+This is equivalent to setting :ts:cv:`proxy.config.ssl.client.verify.server` for this connection.
+
+:code:`NONE` - ``0``
+   Do not verify the upstream server certificate.
+
+:code:`MODERATE` - ``2``
+   Do verification of the upstream certificate but do not enforce.
+
+:code:`STRICT` - ``1``
+   Require successful validation of the upstream certificate to complete the
+   connection.
+
+If ``tunnel_route`` is specified then none of the certificate verification will be done making those
```

Comment:
is specified*,* then none of the certificate verification*s* will be done*,*

Reply:
Fixed.